tests/cli/superset/sync/native/command_test.py

```diff
@@ -1536,6 +1536,87 @@ def test_import_resources_individually_continue(
     assert not Path("progress.log").exists()
 
 
+def test_import_resources_individually_debug(mocker: MockerFixture) -> None:
```

Comment:
Just an observation, we should not be testing against logger calls since that's an implementation detail and are prone to break if we ever change the content of the message, remove it etc.

Reply:
I think your comment is totally fair. I added the test here as well just to ensure future changes won't break this logic -- since this is a CLI, users rely on following the progress in the terminal, so we should make sure details are logged. This was added as a `debug` statement, so it's only going to be visible to customers if they trigger the CLI command with `--loglevel=DEBUG`.